src/System.ServiceModel.Http/src/System.ServiceModel.Http.builds

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.ServiceModel.Http.csproj" >
```

Comment:
I'm not sure the list of variations is correct here. I think we need just net46 and "everything else". On net46, we forward to desktop so don't need any implementation. On everything else we type forward to system.private.servicemodel.